GOVERNANCE.md

```diff
@@ -131,6 +132,15 @@ RFCs are not regarded as documentation. The official specification and implement
 ## Roadmap
 The Cloud Native Buildpacks TOC is responsible for establishing a yearly roadmap laying out our aspirations for that year. This shared vision is essential for keeping the development process focused.
 
+## Vendor Neutrality
+The Cloud Native Buildpacks TOC is committed to ensuring the vendor neutrality of the project in accordance with the [CNCF guidelines on what it means for a project to be vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+
+No company may hold a majority (fifty percent or more) of the total TOC seats through any combination of seats. If the results of an election result in greater than two employees of the same organization, the lowest vote getters from any particular employer will be removed until representation on the committee is down to two from that organization.
```

Comment:
What does it mean by the 'lowest vote getters'?

Reply:
I borrowed that from the Knative community docs:

https://github.com/knative/community/blob/f25ceaff81c265b97d847660e6f2fd2a27d9653e/STEERING-COMMITTEE.md?plain=1#L179

Reply:
> "lowest vote getters"

This is basically the person who receives the least amount of votes.

Reply:
Currently, I assume everyone has equal votes. Should we adapt the wording to our governance model rather than knative?

Reply:
yea, i'll update